.ci/gcb-vcr-nightly.yml

```diff
@@ -0,0 +1,44 @@
+---
+steps:
+    - name: 'gcr.io/graphite-docker-images/go-plus'
+      id: gcb-vcr-nightly
+      entrypoint: '/workspace/.ci/scripts/go-plus/vcr-cassette-update/vcr_cassette_update.sh'
+      secretEnv: ["GITHUB_TOKEN", "GOOGLE_BILLING_ACCOUNT", "GOOGLE_CUST_ID", "GOOGLE_FIRESTORE_PROJECT", "GOOGLE_IDENTITY_USER", "GOOGLE_MASTER_BILLING_ACCOUNT", "GOOGLE_ORG", "GOOGLE_ORG_2", "GOOGLE_ORG_DOMAIN", "GOOGLE_PROJECT", "GOOGLE_PROJECT_NUMBER", "GOOGLE_SERVICE_ACCOUNT", "SA_KEY", "GOOGLE_PUBLIC_AVERTISED_PREFIX_DESCRIPTION"]
+      args:
+          - $BUILD_ID
+
+# Long timeout to enable waiting on VCR test
+timeout: 20000s
+options:
+    machineType: 'N1_HIGHCPU_32'
+
+availableSecrets:
```

Comment:
This should probably set a specific logsBucket similar to https://github.com/GoogleCloudPlatform/magic-modules/pull/9680 - maybe even the same logsBucket? attn @ScottSuarez

Reply:
Or maybe it would make more sense to merge this as-is and clean up later?

Reply:
I can add logsBucket in a later PR + I'm thinking that we can add sweepers in these nightly runs

Reply:
Yeah it would be nice to use a custom service account for this since it's not implemented yet. I can sync with you if you need help.

Otherwise I could adapt this to the new paradigm next week?

Reply:
switched it to use a custom service account.

Reply:
Thanks Shuya!